<?xml version="1.0"?>
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="potential_field">
                <param name="kAtt" value="1.0" />
                <param name="kRep" value="4.0" />
                <param name="dist_thresh" value="2.5" />
                <param name="max_angular_velocity" value="0.5" />
            </group>
        </group>
    </group>
</launch>
